### PR TITLE
Use one decimal instead of two

### LINKED ISSUE #569: Round all content analysis values similarly
We should make sure all content analysis values are rounded similarly: 
> 9.3% of the sentences is written in the passive voice, which is within the recommended range. 

> 2.73% of the words contain over 3 syllables, which is less than or equal to the recommended maximum of 10%.

### PR DESCRIPTION
**Pull request by andizer**
I'd change the decimals from two into one and fixFloatingPoint has been renamed into formatNumber

Fixes #569

### CODE REVIEW

File: spec/assessments/keywordDensitySpec.js [hunk 1]
```diff
@@ -10,22 +10,22 @@ describe( "An assessment for the keywordDensity", function(){
 		var paper = new Paper( "string without the key", {keyword: "keyword"} );
 		var result = keywordDensityAssessment.getResult( paper, Factory.buildMockResearcher( 0.0 ), i18n );
 		expect(result.getScore()).toBe( 4 );
-		expect(result.getText()).toBe( "The keyword density is 0.0%, which is a bit low; the focus keyword was found 0 times.");
+		expect(result.getText()).toBe( "The keyword density is 0%, which is a bit low; the focus keyword was found 0 times.");
```
Comment (jcomack) [inline spec/assessments/keywordDensitySpec.js]: Missing spaces between `(` and `)`.

File: spec/assessments/keywordDensitySpec.js [hunk 2]
```diff
@@ -10,22 +10,22 @@ describe( "An assessment for the keywordDensity", function(){
...
 
 		paper = new Paper( "string with the keyword", {keyword: "keyword"} );
 		result = keywordDensityAssessment.getResult( paper, Factory.buildMockResearcher( 10.0 ), i18n );
 		expect(result.getScore()).toBe( -50 );
-		expect(result.getText()).toBe( "The keyword density is 10.0%, which is way over the advised 2.5% maximum; the focus keyword was found 1 times.");
+		expect(result.getText()).toBe( "The keyword density is 10%, which is way over the advised 2.5% maximum; the focus keyword was found 1 times.");
```
Comment (jcomack) [inline spec/assessments/keywordDensitySpec.js]: Missing spaces before `)`.

File: spec/assessments/keywordDensitySpec.js [hunk 3]
```diff
@@ -10,22 +10,22 @@ describe( "An assessment for the keywordDensity", function(){
...
 
 		paper = new Paper( "string with the keyword", {keyword: "keyword"} );
 		result = keywordDensityAssessment.getResult( paper, Factory.buildMockResearcher( 2.0 ), i18n );
 		expect(result.getScore()).toBe( 9 );
-		expect(result.getText()).toBe( "The keyword density is 2.0%, which is great; the focus keyword was found 1 times.");
+		expect(result.getText()).toBe( "The keyword density is 2%, which is great; the focus keyword was found 1 times.");
```
Comment (jcomack) [inline spec/assessments/keywordDensitySpec.js]: Missing spaces between `(` and `)`.

File: spec/assessments/keywordDensitySpec.js [hunk 4]
```diff
@@ -10,22 +10,22 @@ describe( "An assessment for the keywordDensity", function(){
...
 
 		paper = new Paper( "string with the keyword  and keyword ", {keyword: "keyword"} );
 		result = keywordDensityAssessment.getResult( paper, Factory.buildMockResearcher( 3.0 ), i18n );
 		expect(result.getScore()).toBe( -10 );
-		expect(result.getText()).toBe( "The keyword density is 3.0%, which is over the advised 2.5% maximum; the focus keyword was found 2 times.");
+		expect(result.getText()).toBe( "The keyword density is 3%, which is over the advised 2.5% maximum; the focus keyword was found 2 times.");
```
Comment (jcomack) [inline spec/assessments/keywordDensitySpec.js]: Missing spaces between `(` and `)`.

File: spec/helpers/formatNumberSpec.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+var formatNumberSpec = require( "../../js/helpers/formatNumber.js" );
+
+describe("a test to round numbers to two decimal places", function(){
```
Comment (jcomack) [inline spec/helpers/formatNumberSpec.js]: Missing spaces after `(`.

**Comment by jcomack** [comment]
CR done 👷 

Just some inconsistencies in the tests that need to be addressed. Please also fix the branch conflict :)

**Comment by terw-dan** [comment]
Please fix the errors in the build

**Comment by terw-dan** [comment]
CR Done :+1:

**Comment by terw-dan** [comment]
Acceptance :+1:

**Pull request merged**